com.usebottles.bottles.yml

```diff
@@ -381,8 +381,8 @@ modules:
     sources:
       - type: git
         url: https://github.com/bottlesdevs/Bottles
-        tag: 2022.2.28-trento-2
-        commit: ffda7ef4a84e9c22ad8a2934a157adbee2f10215
+        # tag: 2022.2.28-trento-2
+        commit: a141c51c74ef764635f06cec41c74e6b89ef314f
```

Comment:
we should use the beta branch or restore this in 2022.2.14

Reply:
I don't mind using the beta branch.

Reply:
The latter sounds better.

Reply:
> we should use the beta branch or restore this in 2022.2.14

I think it'd be better if we could create a new release instead? It's easier for packaging and keeps releases more consistent.

Reply:
Ok I’ll publish a new release tomorrow